packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -3756,10 +3736,11 @@ class MetaData extends SingleChildRenderObjectWidget {
   }
 
   @override
-  void debugFillDescription(List<String> description) {
-    super.debugFillDescription(description);
-    description.add('behavior: $behavior');
-    description.add('metaData: $metaData');
+  void debugFillProperties(List<DiagnosticsNode> description) {
+    super.debugFillProperties(description);
+    description.add(new EnumProperty<HitTestBehavior>('behavior', behavior));
+    // TODO(jacobr): this property should be hidden if it is null.
```

Comment:
Removed the TODO. This comment was before I grokked why you wanted to display a number of null properties for debugging purposes any time null was an unexpected state.
I assume you are probably doing something wrong or at least surprising if you have a MetaData widget with no MetaData.